UPGRADE.md

```diff
@@ -199,6 +199,14 @@ These services will be generated automatically based on subject name.
 
 ### ShopBundle
 
+* The following templates were moved:
+
+  * `@SyliusShop/Homepage/_header.html.twig` -> `@SyliusShop/_header.html.twig`
+  * `@SyliusShop/Homepage/_footer.html.twig` -> `@SyliusShop/_footer.html.twig`
+  * `@SyliusShop/Homepage/Menu/_security.html.twig` -> `@SyliusShop/Menu/_security.html.twig`
```

Comment:
These three above are not connected in any way with homepage, they are parts of default shop layout.